index.html

```diff
@@ -21,382 +21,453 @@
+                      alt="Javascript" class="grid-item-icon"/>
+                    <label class="grid-item-text">HTML</label>
+                  </div>
+                  <div class="grid-container">
+                    <img src="assets/images/icons/Ellipse 1-2.png" 
+                      alt="Javascript" class="grid-item-icon"/>
+                    <label class="grid-item-text">CSS</label>
+                  </div>
                 </div>
-                <div class="grid-container">
-                  <img src="assets/images/icons/Ellipse 1-1.png" 
-                    alt="Javascript" class="grid-item-icon"/>
-                  <label class="grid-item-text">HTML</label>
+              </div>
+            </li>
+            <li class="stack-area">
+              <div>
+                <div class="stack-btn">
+                  <h5 class="stack-title">Frameworks</h5>
+                  <img src="assets/images/icons/arrowright.png" 
+                    alt="dropdown icon" class="stack-img"/>
                 </div>
-                <div class="grid-container">
-                  <img src="assets/images/icons/Ellipse 1-2.png" 
-                    alt="Javascript" class="grid-item-icon"/>
-                  <label class="grid-item-text">CSS</label>
+                <div class="languages hidden">
+                  <div class="grid-container">
+                  </div>
                 </div>
               </div>
-            </div>
-          </li>
-          <li class="stack-area">
-            <div>
-              <div class="stack-btn">
-                <h5 class="stack-title">Frameworks</h5>
-                <img src="assets/images/icons/arrowright.png" 
-                  alt="dropdown icon" class="stack-img"/>
-              </div>
-              <div class="hidden">
-                <div class="grid-container">
+            </li>
+            <li class="stack-area">
+              <div>
+                <div class="stack-btn">
+                  <h5 class="stack-title">Skills</h5>
+                  <img src="assets/images/icons/arrowright.png" 
+                    alt="dropdown icon" class="stack-img"/>
                 </div>
-              </div>
-            </div>
-          </li>
-          <li class="stack-area">
-            <div>
-              <div class="stack-btn">
-                <h5 class="stack-title">Skills</h5>
-                <img src="assets/images/icons/arrowright.png" 
-                  alt="dropdown icon" class="stack-img"/>
-              </div>
-              <div class="hidden">
-                <div class="grid-container">
+                <div class="languages hidden">
+                  <div class="grid-container">
+                  </div>
                 </div>
               </div>
-            </div>
-          </li>
-        </ul>
+            </li>
+          </ul>
+        </div>
       </div>
     </section>
     <section class="contact-section">
```

Comment:
- [ ] Kindly consider centering the `Get in Touch` button in this contact section as you can see in the figma design because our design needs to be identical to the figma design.
your design
![v](https://github.com/chudisoft/Portfolio-Microverse/assets/96683843/187bd7be-22b3-4e06-80ea-5e598b2ff19c)

figma design
![f](https://github.com/chudisoft/Portfolio-Microverse/assets/96683843/e047e350-9d0f-4305-8a7b-65900f0b7b55)